functions-and-operators/locking-functions.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Locking Functions
+summary: Learn about user-level locking functions in TiDB.
+---
+
+# Locking Functions
+
+TiDB supports most of the user-level [locking functions](https://dev.mysql.com/doc/refman/5.7/en/locking-functions.html) available in MySQL 5.7.
```

Comment:
```suggestion
TiDB supports locking functions to allow users to explicitly control what things are allowed to execute at the same time.
```

I think this should be a general description of the functionality, especially for users that may not have used MySQL before.

Reply:
I copied the format from other function-reference pages, which links to the MySQL docs here. For example: https://docs.pingcap.com/tidb/stable/string-functions and https://docs.pingcap.com/tidb/stable/numeric-functions-and-operators

I think your suggestion is an improvement, but in the interest of consistency I think we should keep it as it is.